### ISSUE TITLE
Compatibility with the new billing APIs

### ISSUE THREAD

**Issue by drimeisis** [blue-yonder/azure-cost-mon#21]
As Microsoft has retired the old and switched to new billing APIs (https://docs.microsoft.com/en-us/azure/billing/billing-enterprise-api), are there any plans to introduce compatibility with them?

**Comment by MathMagique** [comment]
Indeed, there are plans to do so, and also to include reserved instance consumption. I'll try to make the PR this month.

**Comment by nhjiejan** [comment]
hey @MathMagique , any update on that PR? thanks

**Comment by MathMagique** [comment]
Unfortunately, I have not even come remotely close to working on this, and just looking at my private calendar, this is unlikely to change soon.

**Comment by nhjiejan** [comment]
alright thanks for the update!

**Comment by bhozar** [comment]
@MathMagique does this prevent the cost monitor from working, or will it break in the future until the PR is completed?
